### PR TITLE
Fix GCR reference

### PR DESCRIPTION
**Pull request by kyessenov**

### CODE REVIEW

File: endpoints/getting-started/container-engine.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ spec:
       containers:
       # [START esp]
       - name: esp
-        image: b.gcr.io/endpoints/endpoints-runtime:0.3
+        image: gcr.io/endpoints-release/endpoints-runtime:1
```
Comment (tmatsuo) [inline endpoints/getting-started/container-engine.yaml]: Is it possible to use `latest` tag?

What happens if you release the next image with the tag `2`? Should we update our sample?

Comment (kyessenov) [inline endpoints/getting-started/container-engine.yaml]: We do not have "latest" tag. :1 tracks 1.x.y, :1.0 track 1.0.x, and so on.
Next major release ":2" may not be backwards-compatible.

Comment (tmatsuo) [inline endpoints/getting-started/container-engine.yaml]: makes sense

**Comment by tmatsuo** [comment]
Re: test failure

The test is broken from PRs from other repos. I will create a new PR from the same repo (with the administrator's power) and we'll see the test result there.

**Comment by tmatsuo** [comment]
@kyessenov I just created #283

**Comment by tmatsuo** [comment]
The code LGTM, so I'll merge #283 once the test passes

**Comment by kyessenov** [comment]
Thanks for taking care of the test!

**Comment by tmatsuo** [comment]
Closing in favor of #283

**Pull request closed without merging**